### PR TITLE
Added minimal web UI

### PR DESCRIPTION
**Pull request by alexey-milovidov**
Changelog category (leave one):
- New Feature

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Added minimal web UI to ClickHouse.

### CODE REVIEW

**Comment by alexey-milovidov** [comment]
TODO: add an option to clamp settings instead of throw in HTTP interface.

**Comment by alexey-milovidov** [comment]
TSan: https://github.com/ClickHouse/ClickHouse/pull/15984

File: programs/server/play.html [hunk 1]
```diff
@@ -0,0 +1,437 @@
+<html> <!-- TODO If I write DOCTYPE HTML something changes but I don't know what. -->
```
Comment (adrian17) [inline programs/server/play.html:1]: https://stackoverflow.com/a/20133113/2468469

File: programs/server/play.html [hunk 2]
```diff
@@ -0,0 +1,437 @@
...
+            /// Safety settings to prevent results that browser cannot display.
+            '&max_result_rows=1000&max_result_bytes=10000000&result_overflow_mode=break';
+
+        var query = document.getElementById('query').value;
+        var xhr = new XMLHttpRequest;
+
+        xhr.open('POST', url, true);
+        xhr.send(query);
+
+        xhr.onreadystatechange = function()
```
Comment (adrian17) [inline programs/server/play.html:300]: I'm curious, any particular reason you're mixing modern CSS features (`var`) with old IE-style JS? (XHR+`onreadystatechange`, `for..in`, `event.charCode` etc)

Comment (alexey-milovidov) [reply programs/server/play.html:300]: I "learned" HTML, JS and CSS during my middle-school years in early 2000s (actually never read a book).
I continue to mix my old habits with copy-pasting code from Stackoverflow.

Comment (alexey-milovidov) [reply programs/server/play.html:300]: I tried to use Fetch API but it was not working from `file://` page (most likely due to cross-origin policy, but setting parameter to disable it did not help).

Comment (alexey-milovidov) [reply programs/server/play.html:300]: I have intention to present this simple page as an example of small self-contained code, so everyone can suggest changes to help me to make it perfect from every kind of sense.

File: programs/server/play.html [hunk 3]
```diff
@@ -0,0 +1,437 @@
...
+        var tbody = document.createElement('tbody');
+        for (var row_idx in response.data) {
+            var tr = document.createElement('tr');
+            for (var col_idx in response.data[row_idx]) {
+                var td = document.createElement('td');
+                var cell = response.data[row_idx][col_idx];
+                var is_null = (cell === null);
+                var content = document.createTextNode(is_null ? 'ᴺᵁᴸᴸ' : cell);
+                td.appendChild(content);
+                td.className = response.meta[col_idx].type.match(/^(U?Int|Decimal|Float)/) ? 'right' : 'left';
```
Comment (alexey-milovidov) [inline programs/server/play.html:379]: Suboptimal, will change.

**Comment by vkingnew** [comment]
just use clickhouse server ip to use ，like http://ip:8123/play

**Comment by vkingnew** [comment]
the window is small,50% of the page is better, and Could you provide  SQL formatter？

**Comment by bgranvea** [comment]
nice! the first limitations that I notice:

- cannot select a default database (USE xxx => no session error, if I change URL to http://localhost:8123?database=xxx, I get an error)
- only one statement....

**Pull request merged**